ppdet/modeling/neck/__init__.py

```diff
@@ -1,3 +1,19 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 from . import fpn
+from . import yolo_fpn
```

Comment:
keep the file name same as class name? YOLOFPN or yolov3_fpn?

Reply:
`yolo_fpn.py` define  a module of network structure, such as `yolo_head.py` & `yolo_loss.py` , I think keep name as `yolo_fpn` `yolo_head` `yolo_loss` is ok, and `yolo_fpn` can have multiple implements, like `YOLOv3FPN` `YOLOv4FPN` `PPYOLOFPN` for scalability